saleor/graphql/account/mutations/staff/customer_create.py

```diff
@@ -38,3 +52,70 @@ class Meta:
                 description="Setting a new password for the account is requested.",
             ),
         ]
+
+    @classmethod
+    @traced_atomic_transaction()
+    def save(cls, info: ResolveInfo, instance, cleaned_input):
+        manager = get_plugin_manager_promise(info.context).get()
+
+        default_addresses = cls.save_and_apply_addresses_from_input(
+            cleaned_input=cleaned_input, user_instance=instance
+        )
+
+        super().save(info, instance, cleaned_input)
+
+        if default_addresses["billing"]:
+            instance.addresses.add(default_addresses["billing"])
+        if default_addresses["shipping"]:
+            instance.addresses.add(default_addresses["shipping"])
+
+        instance.search_document = prepare_user_search_document_value(instance)
+        instance.save(update_fields=["search_document", "updated_at"])
```

Comment:
No need for saving the instance twice. Probably we can only call `instance.save()` here instead of calling it in line 65.